### PR TITLE
generalize api benchmark script

### PR DESCRIPTION
**Pull request by brussee**
make it run jenkins

### CODE REVIEW

File: scripts/run_jmeter_benchmark_for_jenkins_api.sh [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
-jmeter_file_path="$PWD/gumby/experiments/tribler/api_benchmark/$TEST_RUNNER_FILE"
-
-# jmeter has no output directory option, so CD into this directory and run it in there.
-pushd $OUTPUT_DIR
+# jmeter has no output directory option, so CD into this directory and run it in there
+cd ${OUTPUT_DIR}
 
 # Run jmeter headless (-n option) and specify the file using -t
-jmeter -n -t $jmeter_file_path
+jmeter -n -t ${EXPERIMENT_DIR}
```
Comment (lfdversluis) [inline scripts/run_jmeter_benchmark_for_jenkins_api.sh:7]: This still needs the `$TEST_RUNNER_FILE` right?

Comment (brussee) [reply scripts/run_jmeter_benchmark_for_jenkins_api.sh:7]: Nope, the config file in the directory defines that.

Comment (lfdversluis) [reply scripts/run_jmeter_benchmark_for_jenkins_api.sh:7]: I think you're thinking about something else. I mean the `.jmx` file a.k.a the test plan for jmeter

Comment (brussee) [reply scripts/run_jmeter_benchmark_for_jenkins_api.sh:7]: `${EXPERIMENT_DIR}` in this case `api_benchmark` which contains `run_requests_benchmark.conf` which defines `test_runner_file = request_benchmark_tribler_api.jmx`
https://github.com/brussee/gumby/blob/c2937ba0698b014076641653c8f25d8c85c95804/experiments/tribler/api_benchmark/run_requests_benchmark.conf

**Pull request closed without merging**